bower.json

```
@@ -5,10 +5,11 @@
     "main.js"
   ],
   "dependencies": {
-    "o-typography": "^5.4.2",
-    "o-grid": "^4.3.7",
-    "o-colors": "^4.1.5",
-    "o-icons": "^5.6.0",
+    "o-typography": "^6.0.0",
+    "o-editorial-typography": "1.0.0-beta.2",
```

Thought a suggestion might be a useful way to get this done. Can be committed once editorial-typography is released.
```suggestion
    "o-editorial-typography": "^1.0.0",
```